magmartfront/src/app/admin/users/create/page.tsx

```diff
@@ -0,0 +1,154 @@
+"use client"
+
+// components/Cadastro.js
+
+import React,{ useState } from 'react';
+import styles from '@/styles/Cadastro.module.css';
+import "@/app/globals.css";
+import axios from 'axios'; // Importe o Axios para fazer requisições HTTP
+import AdmNavbar from '@/components/AdmNavbar';
+
+export default function Cadastrar() {
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [phone, setPhone] = useState('');
+    const [birthDate, setBirthDate] = useState('');
+    const [password, setPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+  
+    const handleCadastro = async (e: any) => {
+      e.preventDefault();
+  
+      // Verifique se as senhas coincidem
+      if (password !== confirmPassword) {
+        console.error('As senhas não coincidem');
+        return;
+      }
+
+
+  
+      // Dados a serem enviados para o backend
+      const userData = {
+        name: name,
+        email: email,
+        phone: `+55`+phone,
+        birth_date: new Date(birthDate),
+        password: password,
+        confirm_password: confirmPassword,
+      };
+  
+      try {
+        // Faça uma requisição POST para a rota de criação de usuário no backend
+        const response = await axios.post('http://localhost:3001/users/new', userData);
```

Comment:
Padronizar a chamada usando variável de ambiente.
```suggestion
        const response = await axios.post('${process.env.API_URL}/users/new', userData);
```